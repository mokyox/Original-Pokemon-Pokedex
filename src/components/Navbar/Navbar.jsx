import React from "react";
import { handleSubmit } from "../utils";

const Navbar = ({ searchChange }) => {
    return (
        <div>
            <nav className="navbar navbar-dark bg-dark fixed-top">
                <h1>Pokedex</h1>
                <form className="form-inline" onSubmit={handleSubmit}>
                    <input
                        className="form-control mr-sm-2"
                        type="search"
                        placeholder="Search Pokemon"
                        aria-label="search"
                        onChange={searchChange}
                    />
                </form>
            </nav>
        </div>
    );
};

export default Navbar;
