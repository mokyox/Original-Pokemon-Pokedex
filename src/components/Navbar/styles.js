import styled from "styled-components";
import { device } from "../../styles/media";

export const PageNavbar = styled.nav`
    padding: 1rem;
    p {
        font-size: 1.5rem;
        font-weight: 600;
        @media ${device.tablet} {
            font-size: 1.75em;
        }
        input {
            padding: 1px;
        }
    }
`;
