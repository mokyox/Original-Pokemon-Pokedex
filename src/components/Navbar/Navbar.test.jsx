import React from "react";
import Navbar from "./Navbar";
import renderer from "react-test-renderer";

describe("<Navbar/>", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<Navbar />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
