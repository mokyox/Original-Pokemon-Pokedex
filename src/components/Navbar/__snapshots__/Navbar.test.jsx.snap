// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar/> renders correctly 1`] = `
<nav
  className="sc-bdfBwQ cBKQTK navbar navbar-dark bg-dark fixed-top justify-content-between"
>
  <p>
    Pokedex
  </p>
  <form
    className="form-inline"
    onSubmit={[Function]}
  >
    <input
      aria-label="search"
      className="form-control mr-sm-2"
      placeholder="Search Pokemon"
      type="search"
    />
  </form>
</nav>
`;
